### ISSUE TITLE
$Solgit  C6FSRsnAKeu7jSGecQTuaVgBzWkBKceh74WHEJTCpump

### ISSUE THREAD

**Issue by Githubmemecoins** [Githubmemecoins/Solgit#1]
The first ever token on launched on the Official Solana GitHub Repo. All communcation will be made within this thread.

C6FSRsnAKeu7jSGecQTuaVgBzWkBKceh74WHEJTCpump

**Comment by vanuzx** [comment]
Let's do it, we can work together!

**Comment by Githubmemecoins** [comment]
Invite more people possible here, send this coin to the moon

**Comment by Githubmemecoins** [comment]
New page open, send it https://github.com/Githubmemecoins/Solgit/issues/1

**Comment by Githubmemecoins** [comment]
> Let's do it, we can work together!

We can do it, top holders still holding

**Comment by Githubmemecoins** [comment]
![OIG4 (93)](https://github.com/user-attachments/assets/5ac3c321-7f6a-4edb-9db7-17ee705855f6)

**Comment by savagex07** [comment]
Send it

**Comment by Githubmemecoins** [comment]
> Send it

Bro wee need to bring here all old community members

**Comment by Grantnav** [comment]
Why was the other one closed

**Comment by Githubmemecoins** [comment]
> Why was the other one closed

I dont know bro...i think the dev not was the creator of the threads, can we keep pushing? We need our lost soldiers here

**Comment by hopiumaddict** [comment]
This is such a banger idea cto on GitHub repo 🤣 send it

**Comment by Githubmemecoins** [comment]
Yes bro but we need old holders here, the whale still holding, if we show that we are a strong community we can do it

**Comment by Githubmemecoins** [comment]
https://t.me/+IViYfqmRFDw2YzE0 join admin chat

**Comment by Githubmemecoins** [comment]
Share it everywhere, we will keep raiding, give me 1 h , im driving

**Comment by Githubmemecoins** [comment]
RAID https://x.com/h2ocoinproject3/status/1838600769185960418?t=vuB8npSnzCWwmKhL7dek9w&s=19

**Comment by Githubmemecoins** [comment]
Raid this 
https://x.com/h2ocoinproject3/status/1838616382491013624?t=ThVY1_WsLdeGBDoglhe3Gw&s=19

**Comment by Githubmemecoins** [comment]
The original rep was closed, whale @idrawline  still holding , join the new rep

https://github.com/Githubmemecoins/Solgit/issues/1

**Comment by hopiumaddict** [comment]
This will send randomly be ready

**Comment by Githubmemecoins** [comment]
> This will send randomly be ready

We just need keep working and bring people here

**Comment by Githubmemecoins** [comment]
> Why was the other one closed

Because the old dev was an idiot

**Comment by Githubmemecoins** [comment]
> Send it

How we can spam with github community?
